src/sif/autocost.cc

```diff
@@ -1546,22 +1525,14 @@ void testAutoCostParams() {
     }
 
     tester = make_autocost_from_json(key, value, extra_json);
-    auto actual = tester->flow_mask_;
-
-    if (actual != expected)
-      throw std::runtime_error("flow_mask_ is not the expected value in test case " +
```

Comment:
if this test were to fail. what message would come out? i presume it would be something about how they two values arent equal but, will it say anything about the flow_mask? im imagining that because its a macro, it might be able to but just wondering.

Reply:
It will look like this
```
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from AutoCost
[ RUN      ] AutoCost.testAutoCostParams
/home/mskurydin/dev/valhalla/src/sif/autocost.cc:1529: Failure
Expected equality of these values:
  tester->flow_mask_
    Which is: '\x10' (16)
  expected
    Which is: '\xF' (15)

```